src/rich-text-container.component.js

```diff
@@ -9,6 +9,14 @@ export function RichTextContainer(props) {
   const newHTMLListenersRef = useRef([])
   const serializers = useRef([])
 
+  useEffect(() => {
+    const contentEditableElement = contextValue.getContentEditableElement();
+
+    if (contentEditableElement && contentEditableElement.innerHTML) {
+      contextValue.fireNewHTML();
+    }
+  }, [contextValue])
```

Comment:
Is the reason why this fixes things that RichTextContainer is that the parent component for all the control buttons and the editor, so by the time the RichTextContainer is mounted all the other components are already mounted (regardless of order)?

Reply:
Yeah, that is my understanding and the premise for the fix.  As I tested things it seemed that child component useEffects run before parent component useEffects based on how React works through the component tree.

Reply:
👍

Reply:
That is my understanding. It seems to me a react component runs all the UEs of children before parent. https://codesandbox.io/s/ypj17kzwxx?fontsize=14